uaclient/yaml.py

```diff
@@ -10,22 +10,27 @@
     import yaml
 except ImportError:
     LOG.error(MISSING_YAML_MODULE.msg)
+    print(MISSING_YAML_MODULE.msg, file=sys.stderr)
```

Comment:
I think it should be fine to use `print`, but I just want to double check if we should change to `event.info` as well